src/FSharp.Compiler.Interactive.Settings/fsihelp.fs

```diff
@@ -0,0 +1,267 @@
+module FSharp.Compiler.Interactive.FsiHelp
+
+[<assembly: System.Runtime.InteropServices.ComVisible(false)>]
+[<assembly: System.CLSCompliant(true)>]
+do ()
+
+open System
+open System.Collections.Generic
+open System.IO
+open System.Text
+open System.Reflection
+
+module Parser =
+
+    open System.Xml
+
+    type Help =
+        {
+            Summary: string
+            Remarks: string option
+            Parameters: (string * string) list
+            Returns: string option
+            Exceptions: (string * string) list
+            Examples: (string * string) list
+            FullName: string
+            Assembly: string
+        }
+
+        member this.ToDisplayString() =
+            let sb = StringBuilder()
+
+            let parameters =
+                this.Parameters
+                |> List.map (fun (name, description) -> sprintf "- %s: %s" name description)
+                |> String.concat "\n"
+
+            sb.AppendLine($"\nDescription:\n%s{this.Summary}") |> ignore
```

Comment:
These could also be something like

```suggestion
            sb.AppendLine()
                .AppendLine("Description:")
                .AppendLine(this.Summary)
                |> ignore
```

(It might be easier to keep track of what pieces have their own line that way.)